tests/forms_tests/tests/__init__.py

```diff
@@ -19,7 +26,8 @@ def wrapper(func):
         def inner(*args, **kwargs):
             for settings in TEST_SETTINGS:
                 with override_settings(**settings):
-                    func(*args, **kwargs)
+                    test_func = skipIf(jinja2 is None, 'this test requires jinja2')(func)
```

Comment:
This will skip Django templates tests also...

Reply:
Yes. Good. 

We need something like: 

```
                    if settings['FORM_RENDERER'] == 'django.forms.renderers.Jinja2':
```

before applying the `skipIf`. 

(It's not very pretty but...)

Reply:
☕

Reply:
This won't work either. The tests run, but they're marked as skipped. @felixxm is adjusting.